components/ws-manager-bridge/src/bridge.ts

```diff
@@ -504,9 +504,27 @@ export class WorkspaceManagerBridge implements Disposable {
             const actuallyRunningInstances = await client.getWorkspaces(ctx, new GetWorkspacesRequest());
             actuallyRunningInstances.getStatusList().forEach((s) => runningInstancesIdx.delete(s.getId()));
 
+            // runningInstancesIdx only contains instances that ws-manager is not aware of
             for (const [instanceId, ri] of runningInstancesIdx.entries()) {
                 const instance = ri.latestInstance;
-                if (instance.status.phase !== "running") {
+                const phase = instance.status.phase;
+                if (phase !== "running") {
+                    // This below if block is to validate the planned fix
```

Comment:
I did it this way because we want to keep the previous check. Therefore I also had to remove the if running check below. Does this make sense? @svenefftinge @geropl

Reply:
perfect! let's see how it does in production 😅